frontend/public/locales/en/translation.json

```diff
@@ -1208,6 +1211,9 @@
   "discoveryConfig.filterform.subheader": "Set filters to discover only relevant clusters.",
   "discoveryConfig.header.add": "Select a credential",
   "discoveryConfig.header.edit": "Select a namespace, then select a credential",
+  "discoveryConfig.InfrastructureProviders.label": "Infrastructure providers",
+  "discoveryConfig.InfrastructureProviders.labelHelp": "All infrastructure providers are included by default unless specified in this drop-down menu.",
```

Comment:
Same request here @oafischer. We should fix this one, and also the original one about the versions "All Red Hat OpenShift versions are included by default unless specified in this drop-down menu."

Reply:
If I'm understanding correctly and it's the same as the previous, then:

_By default, all infrastructure providers are included. Use the drop-down menu to select only specific providers._